pkg/trace/go.mod

```diff
@@ -104,10 +104,15 @@ require (
 	github.com/yusufpapurcu/wmi v1.2.4 // indirect
 	go.opentelemetry.io/auto/sdk v1.1.0 // indirect
 	go.opentelemetry.io/collector/component/componentstatus v0.126.0 // indirect
+	go.opentelemetry.io/collector/confmap v1.32.1-0.20250523042642-a867641d12bd // indirect
+	go.opentelemetry.io/collector/confmap/xconfmap v0.126.1-0.20250523042642-a867641d12bd // indirect
+	go.opentelemetry.io/collector/consumer/consumererror v0.126.1-0.20250523042642-a867641d12bd // indirect
 	go.opentelemetry.io/collector/consumer/consumertest v0.126.0 // indirect
-	go.opentelemetry.io/collector/pdata/pprofile v0.126.0 // indirect
+	go.opentelemetry.io/collector/exporter v0.126.1-0.20250523042642-a867641d12bd // indirect
+	go.opentelemetry.io/collector/extension v1.32.1-0.20250523042642-a867641d12bd // indirect
+	go.opentelemetry.io/collector/pdata/pprofile v0.126.1-0.20250523042642-a867641d12bd // indirect
 	go.opentelemetry.io/collector/pdata/testdata v0.126.0 // indirect
-	go.opentelemetry.io/collector/pipeline v0.126.0 // indirect
+	go.opentelemetry.io/collector/pipeline v0.126.1-0.20250523042642-a867641d12bd // indirect
```

Comment:
These otel versions do not look quite right - we would use released versions here. You may want to wait for https://github.com/DataDog/datadog-agent/pull/37458 to go in first

cc @liustanley

Reply:
Just merged https://github.com/DataDog/datadog-agent/pull/37458 in case this was a blocker